### DISCUSSION TITLE
Simple translation into Perl

### DISCUSSION THREAD

**Discussion by gnustavo** [FuLoRi/Analyze-LastPassVaultGUI#19] [Show and tell]
Hi there! I was unable to run your script in my Ubuntu machine. I installed Powershell but it complained about missing DLLs. Since I'm a Windows noob, I took a look at your code and translated it into a Perl script, without the GUI.

I think it may be easier to use by Linux users out there.

The only new feature I implemented was the conversion of timestamps into date/time strings.

Those interested can find it on https://github.com/gnustavo/analyze-lastpass-vault.

Thank you for the original script. It made my job so much easier. :)

**Comment by FuLoRi** [comment]
Thank you for translating it into Perl! :) I love the idea of converting the timestamps into human-readable date/time strings. Perhaps I'll add that feature to the PowerShell version as well.

**Reply by FuLoRi** [reply]
Thanks to @Iczer1, the PS script also now converts timestamps to date/time strings.
